coral-dbt/src/main/resources/macros/coral_macros/default/utils/get_coral_incremental_response.sql

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+{% macro get_coral_incremental_response(sql, table_names) %}
```

Comment:
The response may contain more information than just the modified SQL (ex. modified table names), depending on endpoint design, so I think it might be better to leave this as get_coral_incremental_response.